### PR TITLE
[WIP] Fix versioning for dev packages

### PR DESCRIPTION
**Pull request by Lull3rSkat3r**

### CODE REVIEW

**Comment by Lull3rSkat3r** [comment]
@lbustelo

File: Makefile [hunk 1]
```diff
@@ -17,8 +17,10 @@
 
 .PHONY: help clean clean-dist build dev test system-test test-travis release pip-release bin-release dev-binder .binder-image audit audit-licenses
 
-BASE_VERSION=0.1.0.dev9
-VERSION=$(BASE_VERSION)-incubating
+BASE_VERSION=0.1.0
+DEV_VERSION?=dev9
+PYPI_VERSION=$(BASE_VERSION).$(DEV_VERSION)
```
Comment (lbustelo) [inline Makefile]: This is problematic for when we are ready to release. It is always placing the `.`

File: Makefile [hunk 2]
```diff
@@ -17,8 +17,10 @@
...
+VERSION=$(BASE_VERSION)-$(DEV_VERSION)-incubating
```
Comment (lbustelo) [inline Makefile]: Same problem here... double `-`

**Comment by lbustelo** [comment]
It is strange that it decided to publish our jars under an additional folder...

https://repository.apache.org/content/repositories/snapshots/org/apache/toree/`kernel`/

**Pull request closed without merging**